jams/schemata/namespaces/pattern/jku.json

```diff
@@ -18,6 +18,6 @@
             "required": ["midi_pitch", "morph_pitch", "staff", "pattern_id", "occurrence_id"]
         },
         "dense": false,
-        "description": "Musical pattern information (MIDI pitch associated with a time onset and a duration), including staff height, voice number, occurrence identification, and pattern identification."
+        "description": "Pattern (MIDI pitch), including staff height, voice number, occurrence id, and pattern id."
```

Comment:
`.` at the end

Reply:
What about `MIDI note number` instead of `MIDI pitch`?

Reply:
Trying to keep it short, I think it's clear enough from context.